### DISCUSSION TITLE
20220327 Update: Could not open input file error

### DISCUSSION THREAD

**Discussion by Sagnasty** [rern/rAudio#863] [Fixed Issues]
Hello,

During the 20220327 update, I received the following errors on two separate machines:

![image](https://user-images.githubusercontent.com/76575081/160284357-d143cd01-c84a-4898-9974-b4c951db6700.png)

Jim

3 x Raspberry Pi 3B+ Rev 1.3 > HiFiBerry DAC+ Pro
1 x Raspberry Pi 3B Rev 1.2 > HiFiBerry DAC+ Pro
Synology DS718+

**Comment by rern** [comment]
Fixed.

**Reply by Sagnasty** [reply]
Confirmed fixed in update 202203271 - thanks Rern!

Jim

3 x Raspberry Pi 3B+ Rev 1.3 > HiFiBerry DAC+ Pro
1 x Raspberry Pi 3B Rev 1.2 > HiFiBerry DAC+ Pro
Synology DS718+
